pkg/network/go/asmscan/scan.go

```diff
@@ -115,7 +116,8 @@ func FindARM64ReturnInstructions(data []byte) ([]uint64, error) {
 	for cursor < len(data) {
 		instruction, err := arm64asm.Decode(data[cursor:])
 		if err != nil {
-			return nil, fmt.Errorf("failed to decode ARM 64 instruction at offset %d within function machine code: %w", cursor, err)
+			cursor += 4
+			continue
```

Comment:
can you leave a comment explaining why are you are continuing?

Reply:
:+1: